### PR TITLE
Create make:datagrid command

### PR DESCRIPTION
**Pull request by thebatclaudio**
Close issue #11

### CODE REVIEW

File: src/Console/Commands/MakeDataGrid.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+     * @param string $fields
+     * @return string
+     */
+    protected function makeColumns(string $fields): string
+    {
+        $columns = "";
+        foreach (explode(",", $fields) as $field) {
+            $label = $field;
+            if (Str::contains($field, ":")) {
+                list($label, $field) = explode(":", $field);
```
Comment (daniel-werner) [inline src/Console/Commands/MakeDataGrid.php]: Could you please change  the order to `list($field, $label)`

File: src/Console/Commands/MakeDataGrid.php [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+     * Replace the class name for the given stub.
+     *
+     * @param string $stub
+     * @param string $name
+     * @return string
+     */
+    protected function replaceClass($stub, $name): string
+    {
+        $className = $this->argument('name');
+        $model = $this->option("model");
```
Comment (daniel-werner) [inline src/Console/Commands/MakeDataGrid.php:41]: Would it be possible to generate a use statement to the top of the created datagrid file for the given model with the default namespace, eg: use App\Models\{model given as argument}?

**Review by daniel-werner — CHANGES_REQUESTED** [review]
Thanks for the PR, works great! I'd like to change some minor things, if you could address those.

**Comment by thebatclaudio** [comment]
Hello @daniel-werner, 
I just made the changes that you requested! Let me know if it's ok

**Comment by daniel-werner** [comment]
@thebatclaudio Changes are fine, merging it. Thanks!

**Pull request merged**